.gitignore

```diff
@@ -1,5 +1,14 @@
+/target
+/work
+/build

```

Comment:
hm? What is this directory? Is it IDEA-specific?

Reply:
copy/paste from another project, removed